sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs

```diff
@@ -44,7 +44,7 @@ internal DevicesClient(RegistryManagerRestClient registryManagerClient, TwinRest
         /// Create or update a device identity.
         /// </summary>
         /// <param name="deviceIdentity">the device identity to create.</param>
-        /// <param name="precondition">The condition on which to perform this operation. To create a device identity, this value must be equal to <see cref="IfMatchPrecondition.Unconditional"/>.</param>
+        /// <param name="precondition">The condition on which to perform this operation. To create a device identity, this value must be equal to <see cref="IfMatchPrecondition.IfMatch"/>.</param>
```

Comment:
I am confused, why do I need to pass in the etag for create operation?

Reply:
Azad and I discussed this a bit offline. The idea here is that the resource to create will have a null ETag value, so it order to get the SDK to not send an if-match header, the user should use the "IfMatch" option

Reply:
Now that I've typed this out though, that would seem confusing to a user

Reply:
you don't, it's optional ... but it's merged with update so we have to have the parameter. I will update the comments ...